code_challenges.rb

```diff
@@ -33,9 +48,25 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
-
-
-
+def vowel_killer
+"Rubber Soul".delete "u"
+"Rbber Soul".delete "o"
+"Rbber Sul".delete "u"
+"Rbber Sl".delete "e"
+end
+p vowel_killer
+
+def vowel_killer2
+"Sgt Pepper".delete "e"
+end
+p vowel_killer2
+
+def vowel_killer3
+"Abbey Road".delete "A"
+"bbet Road".delete "a"
+"bbet Rod".delete "o"
+end
```

Comment:
how can we create just _one_ function that will accomplish this same output? technically, this does what we are asking as far as the test variables provided. but these methods would never work with any other words. the prompt was to "create a method that takes in a string and removes all the vowels from the string." HINT: you would want to create a method that takes in a string as an argument, and you would then use that `.delete` method within to remove the vowels ;)